cmd/krel/cmd/release_notes.go

```diff
@@ -34,6 +36,11 @@ import (
 	"k8s.io/release/pkg/util"
 )
 
+const (
+	// DraftFilename filename for the release notes draft
+	DraftFilename = "release-notes-draft.md"
```

Comment:
nit: This does not have to be public, right?

Reply:
Yes you're right. I thought it could be exposed but there is no need to have it public. Done